raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -199,18 +216,21 @@ public void initialize(DistributedStateMachine stateMachine) throws IOException
 
         if (quorum.isLeader()) {
             electionTimer.reset(Long.MAX_VALUE);
+            fetchTimer.reset(fetchTimeoutMs);
```

Comment:
Seems these two calls are redundant since we do the same thing in `onBecomeLeader`?